cms/djangoapps/contentstore/views/item.py

```diff
@@ -416,7 +416,7 @@ def xblock_view_handler(request, usage_key_string, view_name):
 
         return JsonResponse({
             'html': fragment.content,
-            'resources': hashed_resources.items()
+            'resources': list(hashed_resources.items())
```

Comment:
in py2 returns simple list which can be json serialized. but in py3 returns odict_items([]) which is not json serializeable so the error occures.